os/arch/arm/src/armv7-r/arm_assert.c

```diff
@@ -918,6 +931,67 @@ static void up_dumpstate(void)
 static void _up_assert(int errorcode) noreturn_function;
 static void _up_assert(int errorcode)
 {
+#ifdef CONFIG_FAULT_MGR
```

Comment:
`_up_assert` is a last progress of assert. Before this, there are `board_crashdump` and `boardctl(BOARDIOC_RESET, 0);`. If any config among those configs is enabled, is this working?

Reply:
Thanks for pointing out about it, boardctl(BOARDIOC_RESET, 0) is not needed when fault manager is enabled, so I will add macro dependency above it.

Reply:
Thanks for pointing out this! I think boardctl(BOARDIOC_RESET, 0); this is not needed in fault manager case, I will add dependency of it over !CONFIG_FAULT_MGR.
Thanks

Reply:
How about `board_crashdump`? It waits serial handshake to give dump data.